src/ui/zcl_abapgit_gui_page_diff.clas.abap

```diff
@@ -1020,4 +1000,19 @@ CLASS zcl_abapgit_gui_page_diff IMPLEMENTATION.
     ENDCASE.
 
   ENDMETHOD.
+
+
+  METHOD is_file_requested.
+
+    IF lines( it_files ) = 0.
+      rv_is_file_requested = abap_true.
+      RETURN.
+    ENDIF.
+
+    READ TABLE it_files WITH KEY file-filename = is_status-filename
```

Comment:
can `path` be added to the  conditions? the table has a unique key which will make the READ fast,

```abap
  TYPES:
    ty_stage_tt TYPE SORTED TABLE OF ty_stage
          WITH UNIQUE KEY file-path file-filename .
```

Reply:
That's right, thanks for the hint. Updated.